src/index.ts

```diff
@@ -28,13 +28,19 @@ export function load(app: Application) {
 			domLocation: 'false',
 		} as versionsOptions,
 	});
```

Comment:
Rather don't introduce a new option value here, but rather extend the mixed value declared above, ie.
```
defaultValue: {
	stable: 'auto',
	dev: 'auto',
	domLocation: 'false',
        rootFile: 'package.json',
} as versionsOptions

```
Use 'package.json' as default, this will simplify things later.